pages/privacy/access/requests/index.jsx

```diff
@@ -20,9 +20,13 @@
  */
 
 import React from "react";
-
+import AuthenticatedRoute from "../../../../src/authentication/AuthenticatedRoute";
 import ConsentShowPage from "../../../../components/pages/privacy/access/requests";
 
 export default function ConsentShow() {
-  return <ConsentShowPage />;
+  return (
+    <AuthenticatedRoute>
+      <ConsentShowPage />
```

Comment:
Maybe rename this component to `AccessRequestPage` ? Might be clearer for future developers.

Reply:
I'll make a separate PR for this as I found that we need to rename several components.